notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb

```diff
@@ -0,0 +1,682 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from datasets import load_dataset
</pre><p><code>datasets</code> should be added to requirements (if used in many notebooks) or to this notebook, because it's not available by default</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/785/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I added to this notebook and another one which use it too</p>